api/DOMException.json

```diff
@@ -105,6 +108,9 @@
             "ie": {
               "version_added": "9"
             },
+            "nodejs": {
+              "version_added": false
```

Comment:
This doesn't seem right, `new DOMException('message').code` is 0 in node v20.12.1.

Reply:
Hmm, not sure what's going on with this test then. Reverting!